plugins/Marketplace/lang/en.json

```diff
@@ -8,7 +8,9 @@
         "AllowedUploadFormats": "You may upload a plugin or theme in .zip format via this page.",
         "Authors": "Authors",
         "Browse": "Browse",
-        "TryFreeTrialTitle": "Try 30 days for free, then",
+        "TryFreeTrialTitle": "Try 30 days for free",
+        "Then": "Then",
```

Comment:
Sorry, but I think I meant it the other way around: Make a single larger string that contains the full sentence, so people can translate it at once instead it being pieced together from out of context snippets that might not make any sense in some languages. For example, you can't assume that those two fragments will always be separated by a comma in every language.

see also https://developer.matomo.org/guides/translations#dont-try-to-reduce-the-amount-of-translation-keys

Sorry for being very pedantic here, but a feature or notice that is completely incomprehensible to the majority of Matomo users who don't speak English is worse than before as it makes the software look unprofessional.

Reply:
@Findus23 ah, but the whole sentence is like this, the second part is actually a dropdow. 
![image](https://user-images.githubusercontent.com/3783072/147608867-b979d04b-5e2d-45ee-91a9-e3cab8876abf.png)